test/e2e/framework/helpers/k8s/pv_conf.go

```diff
@@ -0,0 +1,127 @@
+/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package k8s
+
+import (
+	v1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+type PvConfig struct {
+	Name         string
+	Labels       map[string]string
+	Capacity     string
+	AccessModes  []v1.PersistentVolumeAccessMode
+	Source       string
+	Path         string
+	NodeAffinity *v1.VolumeNodeAffinity
+	StorageClass string
+}
+
+func InitPersistentVolume(conf PvConfig) (*v1.PersistentVolume, error) {
+	pv := &v1.PersistentVolume{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: conf.Name,
+		},
+		Spec: v1.PersistentVolumeSpec{
+			Capacity: v1.ResourceList{
+				v1.ResourceStorage: resource.MustParse(conf.Capacity),
+			},
+			AccessModes:                   conf.AccessModes,
+			PersistentVolumeReclaimPolicy: v1.PersistentVolumeReclaimRetain,
+			StorageClassName:              conf.StorageClass,
+		},
+	}
+	if conf.Source == "Local" {
```

Comment:
Could we use a constant for this? Or even better, if you already know different types of sources, then create a type for it.